index.html

```diff
@@ -1816,7 +1816,7 @@ <h5>
             <p its-locale-filter-list="zh-hant" lang="zh-hant">當行尾出現結束夾注符號時，可以縮減其末側二分之一個漢字大小的空白。</p>
           </li>
           <li id="id104">
-            <p its-locale-filter-list="en" lang="en"></p>
+            <p its-locale-filter-list="en" lang="en">According to section 5.1.10 of the <cite>General Rules for Punctuation</cite> (GB/T 15834―2011), when one character wide punctuation appears at the end of a line, half a character space can be reduced behind it.</p>
```

Comment:
Although the english is good, I didn't easily catch the meaning here.  Would this be equivalent?

According to section 5.1.10 of the `<cite>`General Rules for Punctuation`</cite>` (GB/T 15834-2011), when a full-width punctuation character appear as the end of a line, a space character following it can be reduced to half-width.

Reply:
OK,  thank you for your opinion. I found there has some translation like `one character width` and `half character width` to describe punctuation's width in this spec, so I use them in here. But I think your translation `a full-width punctuation` and `half-width` is better than me.

> 可以缩减该符号末侧二分之一个汉字大小的空白 
I thought it want to tell us we can reduce half-width in punctuation self to make it align with last row and next row? Maybe not a character following it? So I misunderstand it?

Reply:
@xfq what's your interpretation of the chinese text?

Reply:
I just checked *General Rules for Punctuation § 5.1.10*. The original text says:

> 标点符号排在一行末尾时，若为全角字符则应占半角字符的宽度（即半个字位置），以使视觉效果更美观。

which means:

> When a fullwidth punctuation mark appears at the end of a line, it should occupy the width of a halfwidth character to make the visual effect more beautiful.

Reply:
IIUC the effect should be something like this (I only found [an example](https://dtp-bbs.com/forum/archives/n20140125124707_20140125124708.html) in Japanese):

![6](https://user-images.githubusercontent.com/2863444/62183812-2b9dca80-b38e-11e9-8db5-68a47d56a6f5.png)

Reply:
@xfq Thank you for your reply. Now we know we should reduce half width when punctuation appear at the end of a line. I think the text "末侧" is unnecessary in the Chinese description "可以缩减该符号末侧二分之一个汉字大小的空白".

> According to section 5.1.10 of the <cite>General Rules for Punctuation</cite> (GB/T 15834-2011), when a full-width punctuation character appears at the end of a line, it can be reduced to half a character width.

Here is the new translation, how about it? @r12a

Reply:
I'll have to rely on you for the meaning of the Chinese, but the English looks good.